src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -36,13 +44,25 @@
 (rf/reg-event-fx
  :wallet-connect/on-session-proposal
  (fn [{:keys [db]} [proposal]]
+   (log/info "Received Wallet Connect session proposal: " {:id (:id proposal)})
    {:db (assoc db :wallet-connect/current-proposal proposal)}))
 
 (rf/reg-event-fx
- :wallet-connect/reset-current-session
+ :wallet-connect/on-session-request
+ (fn [_ [event]]
+   (log/info "Received Wallet Connect session request: " event)
+   {:fx [[:dispatch [:wallet-connect/process-session-request event]]]}))
+
+(rf/reg-event-fx
+ :wallet-connect/reset-current-session-proposal
  (fn [{:keys [db]}]
    {:db (dissoc db :wallet-connect/current-proposal)}))
 
+(rf/reg-event-fx
+ :wallet-connect/reset-current-session-request
+ (fn [{:keys [db]}]
+   {:db (dissoc db :wallet-connect/current-request)}))
+
 (rf/reg-event-fx
  :wallet-connect/pair
```

Comment:
FYI: This event has no usages.

Reply:
It's being used in another PR. We had a PR to add the basic functionality for pairing (via the REPL), since pairing is essential in all the flows. The flow that has the UI which uses this event is being reviewed at the moment.

Reply:
It's used in my PR.